tests/tool/tcprelay/main.cpp

```diff
@@ -641,10 +669,94 @@ MegaCLILogger logger;
 // local console
 Console* console;
 
+// input read from user.
+char* line = nullptr;
+
+#ifndef NO_READLINE
+
+char* longestCommonPrefix(ac::CompletionState& acs)
+{
+    string s = acs.completions[0].s;
+    for (size_t i = acs.completions.size(); i--; )
+    {
+        for (unsigned j = 0; j < s.size() && j < acs.completions[i].s.size(); ++j)
+        {
+            if (s[j] != acs.completions[i].s[j])
+            {
+                s.erase(j, string::npos);
+                break;
+            }
+        }
+    }
+    return strdup(s.c_str());
+}
+
+char** my_rl_completion(const char */*text*/, int /*start*/, int end)
+{
```

Comment:
Can we add a space between the first */ characters please?   MSVC complains about end-comment outside of a comment, and the line is showing up red here in the github compare screen.  thanks

Reply:
Sure thing.

Since they're not being used, I'll just leave out the names entirely.